src/compressor.js

```diff
@@ -11,20 +11,17 @@ define(function (require) {
    * and sound production. Compression creates an overall louder, richer, 
    * and fuller sound by lowering the volume of louds and raising that of softs.
    * Compression can be used to avoid clipping (sound distortion due to 
-   * peaks in volume) and is especially useful when many sounds are being used 
+   * peaks in volume) and is especially useful when many sounds are played 
    * at once. Compression can be used on indivudal sound sources in addition
    * to the master output. This class is built using the 
-   * Web Audio Dynamics Compressor Node
+   * Web Audio Dynamics Compressor Node <br>
    * https://www.w3.org/TR/webaudio/#the-dynamicscompressornode-interface
    *
    * @class p5.Compressor
    * @extends p5.Effect
    * @constructor
    *
-   * @example
-   * <div><code>
-   *
-   * </code></div>
```

Comment:
looking forward to a Compressor example, maybe one that focuses on controlling threshold would be easiest to hear / visualize?